i18n.config.js

```diff
@@ -6,8 +6,14 @@ module.exports = {
       name: 'English',
       file: 'en.js',
     },
+    {
+      code: 'hi',
+      iso: 'hi',
+      name: 'हिंदी',
+      file: 'hi.js',
+    },
```

Comment:
just ignore the `i18n.config.js` file, I will do that when the PR is ready :)

Reply:
yes, that's just for development, I'll revert that file to the original commit when the branch happened before I mark this PR for review.